### PR TITLE
debugged 99%, need help with border

### PR DESCRIPTION
**Pull request by PemaYT**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -68,24 +99,23 @@ body {
...
     width: 100%;
     outline: none;
-    border: 1.5px solid;
+    border: 1px solid;
     border-radius: 5px;
     font-size: 17px;
     transition: all 0.3s ease;
     padding: 5px;
-    border-color: rgb(199, 199, 199);  
+    color: rgb(163, 157, 157);
```
Comment (Zyanvel) [inline styles.css:113]: What is the difference between this input box and the error class mentioned above?

File: styles.css [hunk 2]
```diff
@@ -12,17 +13,45 @@ body {
...
 .error-text,
 .error-icon {
     display: none;
 }
+
 .error {
-	border: 2px solid hsl(0, 100%, 74%);
+	border: 3px solid;
+    border-radius: 5px;
+    color: hsl(0, 100%, 74%);
```
Comment (Zyanvel) [inline styles.css:29]: What is the difference between this error class and the border input box mentioned below?

**Review by Zyanvel — COMMENTED** [review]
Left some comments to open a debug dialog

File: styles.css (hunk 2 above)
Comment (PemaYT) [reply styles.css:29]: the error class is from JS, we put a class in JS. hmmm maybe it should be input box?

**Pull request merged**